release-notes.md

```diff
@@ -2,7 +2,13 @@
 
 ## Development
 
-[Commits](https://github.com/kpdecker/jsdiff/compare/v5.0.0...master)
+[Commits](https://github.com/kpdecker/jsdiff/compare/v5.1.0...master)
+
+## v5.1.0
+
+- [#365](https://github.com/kpdecker/jsdiff/issues/365) Allow early termination to limit execution time with degenerate cases
+
+[Commits](https://github.com/kpdecker/jsdiff/compare/v5.0.0...v5.1.0)
```

Comment:
(This wrongly said https://github.com/kpdecker/jsdiff/compare/v5.0.0...v5.0.1 on npm; I guess Kevin changed his mind about whether the new version should be 5.0.1 or 5.1.0 partway through releasing. I've fixed it.)